beamer/agent/models/claim.py

```diff
@@ -163,6 +164,9 @@ def _on_new_claim_made(self, new_claim_made: ClaimMade) -> None:
         self._latest_claim_made = new_claim_made
         self.transaction_pending = False
 
+    def challenger_exists(self) -> bool:
+        return len(self._challenger_stakes.keys()) > 0
```

Comment:
Nitpick: it is customary to prefer the boolean truth value of containers in cases like these, e.g.
```python
return bool(self._challenger_stakes)
```